test/app.router.js

```diff
@@ -648,7 +648,8 @@ describe('app.router', function(){
       .expect('122', done);
     })
 
-    it('should eat everything after /', function(done){
+    // TODO: I don't think this is valid anymore?
+    xit('should eat everything after /', function(done){
```

Comment:
I temporarily disabled this test because I was't sure this was supported in `router-beta.1`. If someone could confirm it is, indeed, not, I'll go ahead and remove it. If it _is_, please let me know how to update the syntax.

Reply:
That is a good question. I'm not sure myself and would need to look to find out. Do you want to do that leg work, or are you asking to defer that to someone else?

Reply:
I was hoping someone else could answer off the top of their heads. That' ok - I'll try to take a look myself. What happens now is that, even after updating the syntax, the `:user*` group still matches everything after `/`.

Reply:
@dougwilson Any reason why `router` would use `path-to-regexp` `3.2.0`? Was that the latest at the time it was released? It's now on `6.2.0`. Would it make sense to target that and would it be too big a leap?

Reply:
So, based on [this test from `pillarjs/router`](https://github.com/pillarjs/router/commit/4acf108bcd52c16e52ce31256e757580398265d9#diff-859f2b16134f796c6f0bb540c54d824024ff05b9722708e13ee78b29476e3471R757) which addresses the same situation, I can confirm the above doesn't work anymore with `path-to-regexp` `3.x`+.

```sh
# path-to-regexp@3.2.0

> const p = require('path-to-regexp')
> p.match('/u/:users*')('/u/nfantone/foo/bar')
{
  path: '/u/nfantone/foo/bar',
  index: 0,
  params: { users: [ 'nfantone', 'foo', 'bar' ] }
}
> p.match('/u/:users(.*)')('/u/nfantone/foo/bar')
{
  path: '/u/nfantone/foo/bar',
  index: 0,
  params: { users: 'nfantone/foo/bar' }
}
```

```sh
# path-to-regexp@0.1.7

> const p = require('path-to-regexp')
> '/u/nfantone/foo/bar'.match(p('/u/:users*'))
['/u/nfantone/foo/bar', 'nfantone', '/foo/bar']
```

What shall we do with the test?

Reply:
> So the test section this is in is *, and I see you changed everything to (.*) instead. Does that change not work for this instance?

Unfortunately, no. As shown in the snippet above it matches all segments until the end of the path (e.g.: `{ users: 'nfantone/foo/bar' }`, instead of just `nfantone`.

> As much as I can remember, this is just what someone helped make a pull request for.

I thought you were the one [releasing `router@2.0.0-beta.1`](https://github.com/pillarjs/router/commit/4acf108bcd52c16e52ce31256e757580398265d9#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)? That's where the dependency to `path-to-regexp@3.2.0` comes from. I think you might be confusing it with some other related change.

> If you believe you can make it even more up to date, a PR would be wlecome in that regard.

I could try and see if `router` works with newer versions of `path-to-regexp`. Seems like it shouldn't be too hard to pull off.

Reply:
> I thought you were the one releasing router@2.0.0-beta.1? That's where the dependency to path-to-regexp@3.2.0 comes from. I think you might be confusing it with some other related change.

Sure, I make releases, but a release contains commits, many of which are pull requests that I did not author. I did not author the upgrade of path-to-regexp. I actually did check :) What other change would I be confusing it with, can you elaborate?

Reply:
>What other change would I be confusing it with, can you elaborate?

Got confused myself, apologies. I even deleted that bit - but your answer got here first.

> Do you think we should finish this first to get a release out and then do that, or delay this until that is done? 

Done already 👉🏼 https://github.com/pillarjs/router/pull/102. Turns out, it just worked out-of-the-box.